cypress/utils/utils.ts

```diff
@@ -101,10 +109,30 @@ export function selectFilter(filterName: string): void {
     cy.get("ul[role=menu] > li").contains("button", filterName).click();
 }
 
-export function applySearchFilter(filterName: string, searchText: string): void {
+export function applySearchFilter(filterName: string, searchText: any): void {
     selectFilter(filterName);
-    inputText(commonView.filterInput, searchText);
-    click(commonView.searchButton);
+    if (filterName == businessservice || filterName == tag) {
+        cy.get("div.pf-c-input-group").find("div.pf-c-select > div > button").click();
+        if (Array.isArray(searchText)) {
```

Comment:
Any specific reason for using array value as search text?

Reply:
Scenarios:
1. Searching multiple items with unique names.
2. Tag and Business service filter have multiple selections. 
Hence enhanced the method.